source/gcp/prerequisites/credentials.rst

```diff
@@ -9,46 +9,81 @@
 Configuring GCP credentials
 ===========================
 
-In order to make the Wazuh GCP module pull log data from Google Pub/Sub, it will be necessary to provide access credentials so it can connect to them.
+In order to make the Wazuh GCP module pull log data from Google Pub/Sub or Google Storage, it will be necessary to provide access credentials so it can connect to them.
```

Comment:
```suggestion
In order to make the Wazuh GCP module pull log data from Google Pub/Sub or Google Storage, it will be necessary to provide access credentials so that it can connect to them.
```